timetracker/vms/templates/vms/employee-dash.html

```diff
@@ -4,25 +4,16 @@
 {% load time_record_tags %}
 
 {% block content %}
-<h2>You are{% if not employee.is_clocked_in %} not{% endif %} currently clocked in.</h2>
-  {% if employee.is_clocked_in %}
-    <a class="btn btn-primary" href="{{ employee.clock_out_url }}">Clock Out {{ employee }}</a>
-  {% else %}
-    <a class="btn btn-primary" href="{{ employee.clock_in_url }}">Clock In {{ employee }}</a>
-  {% endif %}
-
-<h2>Stats</h2>
-  <p></p>
-  <p>{{ request.user.name }}, you have worked {{ total_hours }} hour(s).</p>
-  <button>click me for all stats</button><button>click me for this pay period</button>
-  <p>You are looking at the most recent time period.</p>
-  <p> 0 of the {% for employee in employees %}
-   {{ employee.time_records.all|length }}
-  {% endfor %} time records are approved.</p>
-  <p> You have 0 approved hours out of 20.5 total worked hours this pay period.</p>
-  <p>You have worked an average of 4 hrs/day and 20.5 hrs/week.</p>
-  <p>-----------------------</p>
+  <h2>{{employee.user.name}},</h2>
+  <h2>You are{% if not employee.is_clocked_in %} not{% endif %} currently clocked in{% if employee.is_clocked_in %} as a {{open_time_record.job.name}} for {{employee.staffing_agency}}{% endif %}.
```

Comment:
If the user is the staffing agency manager who hired the employee, they should not see the "clock in/out" information.